### PR TITLE
feat: add footer output option

### PR DESCRIPTION
**Pull request by PengBoUESTC**
### Description

add `footer` output option based on `banner` option

### Test Plan

---

### CODE REVIEW

**Review by underfin — APPROVED** [review]

**Comment by underfin** [comment]
Thank you.

**Comment by PengBoUESTC** [comment]
> Thank you.

thanks!

File: packages/rolldown/tests/fixtures/output/footer/_config.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import type {
+  RollupOptions,
+  RolldownOutput,
+  RolldownOutputChunk,
+} from '../../../../src'
+import { expect } from 'vitest'
```
Comment (hyf0) [inline packages/rolldown/tests/fixtures/output/footer/_config.ts:6]: This should be `import { ... } from 'rolldown'` directly. And you don't need to import these types with using `defineTest`. Types will be inferred automatically.

Comment (PengBoUESTC) [reply packages/rolldown/tests/fixtures/output/footer/_config.ts:6]: Ok, I will fix these .

Comment (hyf0) [reply packages/rolldown/tests/fixtures/output/footer/_config.ts:6]: Nice. I create an [issue](https://github.com/rolldown/rolldown/issues/695) for this. Not just the banner test, some other tests also have this issue.

Comment (PengBoUESTC) [reply packages/rolldown/tests/fixtures/output/footer/_config.ts:6]: Ok

**Pull request merged**